docs/notes.md

```diff
@@ -0,0 +1 @@
+The plugin tries to cache the Vault token obtained from logging into Vault on the `argocd-repo-server`'s container's disk, at `~/.avp/config.json` for the duration of the token's lifetime. This of course requires that the container user is able to write to that path. Some environments, like Openshift 4, will force a random user for containers to run with; therefore this feature will not work, and the plugin will attempt to login to Vault on every run. This can be fixed by ensuring the `argocd-repo-server`'s container runs with the user `argocd`.
```

Comment:
I think this would be better placed in the usage.md page or something rather than standalone but it's not that important to me

Reply:
> like Openshift 4

Oh and it's not just Openshift 4, it's Openshift in general that has SCCs so it'd be more accurate to write "like Openshift"